lib/phone-format-interface.js

```diff
@@ -116,7 +116,19 @@ function isValidNumber(phone, country) {
         var phone = cleanPhone(phone);
         var phoneUtil = i18n.phonenumbers.PhoneNumberUtil.getInstance();
         var number = phoneUtil.parseAndKeepRawInput(phone, country);
-        return phoneUtil.isValidNumber(number);
+        if (typeof type !== 'undefined') {
```

Comment:
If this is NULL it will blow up on the next line when `toUpperCase` is called.  Maybe add a NULL check as well here?